qtawesome/__init__.py

```diff
@@ -81,6 +82,10 @@
 }
 
 
+def install_bundled_fonts_system_wide():
```

Comment:
Here you can find some ideas on how to make this require elevated privileges automatically (in case the user is not an admin):

https://stackoverflow.com/questions/19672352/how-to-run-script-with-elevated-privilege-on-windows